internal/backends/winit/glcontext.rs

```diff
@@ -135,53 +133,57 @@ impl OpenGLContext {
     }
 
     pub fn new_context(
-        window_builder: winit::window::WindowBuilder,
+        _window: &dyn raw_window_handle::HasRawWindowHandle,
+        _display: &dyn raw_window_handle::HasRawDisplayHandle,
+        _size: PhysicalSize,
         #[cfg(target_arch = "wasm32")] canvas_id: &str,
```

Comment:
This is somewhat of a kludge. It would be better if `RawWindowHandle` would have a better way of identifying the canvas element. ALAS the int id in the web handle is local to winit and not unique in the DOM. But when a future version of raw-window-handle hopefully improves this, we can remove this.

Reply:
Is there an upstream issue for that?  Maybe https://github.com/rust-windowing/raw-window-handle/issues/102 ?

Reply:
Yes, that's the one I saw as well. Wasn't sure about creating a backlink heh, but that's done now.

Reply:
But i think that could be good enough actually.
We can set the `data-raw-handle` to some unique value somewhere else. Possibly in our js specific API.
Anyway, this is unrelated to this change.